v4/source/teams.yaml

```diff
@@ -429,6 +429,46 @@
             // Update team's privacy to Private
 
             updatedTeam, resp := Client.UpdateTeamPrivacy(<TEAMID>, model.TEAM_INVITE)
+  "/teams/{team_id}/restore":
+    post:
+      tags:
+        - teams
+      summary: Restore a team
+      description: |
+        Restore a team that was previously soft deleted.
+
+        __Minimum server version__: 5.24
+
+        ##### Permissions
+        Must have the `manage_team` permission.
+      parameters:
+        - name: team_id
+          in: path
+          description: Team GUID
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: Team restore successful
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Team"
+        "401":
+          $ref: "#/components/responses/Unauthorized"
+        "403":
+          $ref: "#/components/responses/Forbidden"
+        "404":
+          $ref: "#/components/responses/NotFound"
+      x-code-samples:
+        - lang: Go
+          source: |
+            import "github.com/mattermost/mattermost-server/model"
```

Comment:
Non-blocking and can be on separate PR if it's true.  Should this (and all other instances) to be updated using `.../v5/model`?

Reply:
Yes, I agree.  We'll have to create a new ticket for that and update all of them.

Reply:
Hi @saturninoabril , I added a ticket to platform backlog to address this.